OData/src/System.Web.OData/OData/Query/Expressions/AggregationBinder.cs

```diff
@@ -264,7 +296,7 @@ private Expression BindAccessor(SingleValueNode node)
                     return CreatePropertyAccessExpression(BindAccessor(propAccessNode.Source), propAccessNode.Property);
                 case QueryNodeKind.SingleValueOpenPropertyAccess:
                     var openNode = node as SingleValueOpenPropertyAccessNode;
-                    return Expression.Property(BindAccessor(openNode.Source), openNode.Name);
```

Comment:
>Expression.Property(BindAccessor(openNode.Source), openNode.Name) [](start = 27, length = 65)

So that I understand this change, it looks like the old code looked for the property on the source, rather than as a member of the dynamic properties collection? Was that just broken? Will changing this code break any existing logic that successfully bound to a property with the name of the dynamic property, rather than looking it up in the collection, or would it not have gotten called anyway?